wordpress-mobile/WordPress-iOS
# Gutenberg Mobile: Emit save event to Gutenberg upon preview

Fix for https://github.com/wordpress-mobile/gutenberg-mobile/issues/5677

## To Test

1. Upload a video to the VideoPress block.
2. Change some of the settings that will be synced (e.g. title, privacy, rating, etc).
3. Tap on the three dots button.
4. Tap on Preview.
5. Close the preview.
6. Open the block settings and confirm that the changed settings have persisted.

## Regression Notes
1. Potential unintended areas of impact

No code has been refactored or changed as part of this PR, only two extra calls to a pre-existing function have been made. As such, I don't foresee unintended areas of impact.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Manual testing and a reliance on existing tests. 

3. What automated tests I added (or what prevented me from doing so)

N/A

<hr />

PR submission checklist:


UI Changes testing checklist:
